CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing Guide
+
+Thank you for investing your time in contributing to our project!
+
+Read our [Code of Conduct](CODE_OF_CONDUCT.md) to keep our community approachable and respectable.
+
+## Issues and Support
+
+If you find a problem or have a question about the content in this repository and how to use it,
+please create an issue in the [Issues Tracker](https://github.com/Onto-Med/GFO/issues).
+Make sure you describe the problem as clearly as possible.
+
+## Workflow to Make Contributions
+
+We are very happy about contributions!
+
+It is recommended to use an ontology editor such as [Protégé](https://protege.stanford.edu/) to modify the ontology [cop.owl](cop.owl).
```

Comment:
`cop.owl` might be a typo, because there is no file here with this name. Did you mean `gfo.owl`?

Reply:
Fixed